test/sql/types/nested/map/test_map_from_entries.test

```diff
@@ -0,0 +1,85 @@
+# name: test/sql/types/nested/map/test_map_from_entries.test
+# description: Test basic functionality of map_from_entries function
+# group: [map]
+
+statement ok
+PRAGMA enable_verification
+
+query I
+select MAP_FROM_ENTRIES(ARRAY[(1, 'x'), (2, 'y')]);
+----
+{1=x, 2=y}
+
+# Empty arguments not allowed
+statement error
+SELECT map_from_entries();
+
+# Provided argument is NULL
+statement error
+SELECT MAP_FROM_ENTRIES(NULL);
+
+# Some of the structs provided are NULL
+statement error
+SELECT MAP_FROM_ENTRIES(ARRAY[NULL, (1, 'x'), NULL, (2, 'y')]);
+
+# Provide column as parameter
+statement ok
+CREATE TABLE t1 AS (SELECT ARRAY[(1, 'x'), (2, 'y'), (4, 's')] AS list);
+
+statement ok
+INSERT INTO t1 VALUES (ARRAY[(2, 'a'), (3,'b')])
+
+statement ok
+INSERT INTO t1 VALUES (ARRAY[(6, 'h'), (7,'g')])
+
+query I
+SELECT MAP_FROM_ENTRIES(list) FROM t1;
+----
+{1=x, 2=y, 4=s}
+{2=a, 3=b}
+{6=h, 7=g}
+
+statement ok
+INSERT INTO t1 VALUES (NULL)
+
+query I
+SELECT MAP_FROM_ENTRIES(list) FROM t1;
+----
+{1=x, 2=y, 4=s}
+{2=a, 3=b}
+{6=h, 7=g}
+{}
```

Comment:
I am not so sure that this should be an empty map? Shouldn't it be `NULL`? That would mean that you need to set this row as `invalid` in the validity mask.